lib/barnes/instruments/puma_backlog.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Barnes
+  module Instruments
+    class PumaBacklog
+      def initialize(sample_rate=nil)
+      end
+
+      def valid?
+        defined?(Puma) &&
+          Puma.respond_to?(:stats) &&
+          ENV["DYNO"].start_with?("web")
+      end
+
+      def start!(state)
+        require 'multi_json'
+      end
+
+      def json_stats
+        MultiJson.load(Puma.stats || "{}")
```

Comment:
So, maybe this just hasn't been updated yet. But, it seems like parsing JSON is not necessary anymore? Is that accurate?

Reply:
I didn't want to introduce `require 'json'` into Puma, so it returns a string of valid json rather than json.

This is still required.

Reply:
Ah. I thought there was a `stats_object` introduced in your PR, but on closer inspection that was a `stats_object=`, a different thing entirely. 

I don't at _all_ understand why https://github.com/puma/puma/blob/master/lib/puma/cluster.rb#L344 generates a string instead of generating an object that is serialized to JSON when necessary, but I'll leave it be. :)